cluster/gce/util.sh

```diff
@@ -836,6 +867,18 @@ function delete-network() {
 
 function delete-subnetworks() {
   if [[ ${ENABLE_IP_ALIASES:-} != "true" ]]; then
+    if [[ "${ENABLE_BIG_CLUSTER_SUBNETS}" = "true" ]]; then
```

Comment:
After my change to line 1658 if we enter this part of code it means that `KUBE_DELETE_NETWORK=true`. This may be wrong, but I'd argue that semantics of  `KUBE_DELETE_NETWORK=false` was "I want to keep my networking setup", and "=true" I want to get rid of all Kubernetes networking. If we want to have more possibilities we need more flags...